### PR TITLE
VueJS London blogpost

### PR DESCRIPTION
**Pull request by ElkeCodes**

### CODE REVIEW

File: _posts/2019-10-07-vuejs-london-2019.md [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+
+With 8 talks, 1 keynote and 4 lightning talks, the conference day promised to be quite busy.
+The conference was held at O2 Cineworld, which is not that far from the city center.
+As a venue this was exquisite as the accommodation was top-notch, the visuals were beautiful, the talks were clearly audible and the food was great.
+
+<p class="align-center">
+  <img src="{{ '/img/vuejs-london-2019/venue.jpg' | prepend: site.baseurl }}" class="image fit" alt="The VueJS London venue">
+</p>
+
+Before noon, a total of 5 talks was planned with topics ranging from animations, styling, visualisations and performance to the future of the Vue router.
```
Comment (miva2) [inline _posts/2019-10-07-vuejs-london-2019.md]: "was planned" -> "were planned"?

File: _posts/2019-10-07-vuejs-london-2019.md [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+A final addition would be the full use of the Composition API so we have functionalities such as `useLink`, `useLocation`, `onBeforeRouteLeave` and `onBeforeRouteUpdate`.
+
+The focus of the new router will be mainly on supporting Vue 3 while smaller improvements will be done for Vue 2.
+
+> [View slides](https://slides.com/posva/a-new-router-to-guide-your-apps){:target="_blank" rel="noopener noreferrer"}
+
+## Lightning talks
+
+### How to get your product owners to write your functional tests, by [Callum Silcock](https://twitter.com/csiilk){:target="_blank" rel="noopener noreferrer"}
+With the help of the [Cypress Cucumber preprocessor](https://github.com/TheBrainFamily/cypress-cucumber-preprocessor){:target="_blank" rel="noopener noreferrer"} Callum was able demonstrate how code like this could be processed by Cypress:
```
Comment (miva2) [inline _posts/2019-10-07-vuejs-london-2019.md]: typo: seems like a "to" is missing.
"was able demonstrate" -> "was able to demonstrate"

Comment (ElkeBxl) [reply _posts/2019-10-07-vuejs-london-2019.md]: Fixed, thx! :)

File: _posts/2019-10-07-vuejs-london-2019.md (hunk 1 above)
Comment (ElkeBxl) [reply _posts/2019-10-07-vuejs-london-2019.md]: Fixed, thx! :)

**Pull request merged**